Tests/StorageApiHandler.php

```diff
@@ -68,18 +52,14 @@ public function getVerbosity()
     }
 
     /**
-     * @inheritdoc
+     * @param array $record
      */
     public function handle(array $record): bool
     {
         if (($this->verbosity[$record['level']] == self::VERBOSITY_NONE) || empty($record['message'])) {
             return false;
         }
 
-        if (!$this->storageApiClient) {
-            return false;
-        }
```

Comment:
Zajímavý, tohle mám zafixovaný, že bylo strašně důležitý, ale nevidím způsob jak by se to mohlo stát, tak snad je to opravdu outdated